src/main.py

```diff
@@ -1,38 +1,35 @@
 import pandas as pd
 
 from job_description_feature_extraction import get_one_hot_encoded_words, get_rake_keywords
-from contract_cleaning import get_one_hot_encoded_contract
+from cleaning_functions import get_one_hot_encoded_feature, get_binary_encoded_feature
 
 train_raw_data_csv_file_name = '../data/Train_rev1.csv'
 test_raw_data_csv_file_name = '../data/Test_rev1.csv'
 
 if __name__ == "__main__":
     train_raw_data = pd.read_csv(train_raw_data_csv_file_name)
-    test_raw_data = pd.read_csv(test_raw_data_csv_file_name)
-    # Train fields
-    train_description_feature = train_raw_data[['FullDescription']]
-    train_con_type_feature = train_raw_data[['ContractType']]
-    train_con_time_feature = train_raw_data[['ContractTime']]
-    # Test fields
-    test_description_feature = test_raw_data[['FullDescription']]
-    test_con_type_feature = test_raw_data[['ContractType']]
-    test_con_time_feature = test_raw_data[['ContractTime']]
```

Comment:
Removed features related to test set since we don't know if we are going to receive results from Adzuna.